src/app/style.css

```
@@ -1,3 +1,17 @@
+.header {
+  padding: 10px;
+}
+
+.show {
+  font-size: 25px;
+  font-family: "Inter";
+  display: flex;
+  justify-content: flex-end;
+  padding: 20px;
+  position: relative;
+  bottom: 60px;
+}
```

Aqui a gente vai mudar um pouquinho o css pra bater com a sugestão de cima.

```suggestion
.page-header {
  display: flex;
  justify-content: space-between;
  align-items: flex-end;
  padding: 24px;
}

.header-user-tag {
  font-size: 25px;
}
```